SmartDeviceLinkTests/RPCSpecs/StructSpecs/SDLImageSpec.m

```diff
@@ -34,49 +36,106 @@
             testSDLImage = [[SDLImage alloc] init];
             testSDLImage.value = value;
             testSDLImage.imageType = imageType;
+            testSDLImage.isTemplate = @YES;
 
             expectedValue = value;
             expectedImageType = imageType;
+            expectedIsTemplate = @YES;
         });
 
-        it(@"Should get correctly when initialized as a dictionary", ^{
+        it(@"Should set and get correctly when initialized", ^{
             NSString *value = @"value";
             SDLImageType imageType = SDLImageTypeStatic;
 
             NSMutableDictionary* dict = [@{SDLNameValue:value,
```

Comment:
Can be regular `NSDictionary` and a no `mutableCopy` now

Reply:
Fixed